### DISCUSSION TITLE
How to set a different height for the top and the bottom plot?

### DISCUSSION THREAD

**Discussion by forna** [highfestiva/finplot#325] [Q&A] [answered]
I have a chart with two plots, the top one renders the candlesticks and the bottom one renders an indicator.
Is it possible to resize them so the top plot takes 80% of the height and the bottom one the remaining 20%?
I have found the methods to crop the Y range and to set a different Y scale but I cannot find a way to set the plot height.
Thank you!

![image](https://user-images.githubusercontent.com/1431006/165965781-fea1311d-287c-4982-bf30-9b1e3171771e.png)

Code:
https://github.com/forna/Finplot_PySide_chart_with_study/blob/master/main.py

**Comment by highfestiva** [comment, marked as answer]
Since you're rolling your own layout, you have to make the layout scale according to your needs. PyQt ppl knows more.

**Reply by forna** [reply]
Oops you are right, I hadn't realized that.
So here is the solution, add the stretch attribute to the addWidget method:
```
self.layout.addWidget(self.fplt_main.ax_widget, stretch=3)
self.layout.addWidget(self.fplt_study.ax_widget, stretch=1)
```

I have updated the code too:
https://github.com/forna/Finplot_PySide_chart_with_study/blob/master/main.py

Thanks!
